### PR TITLE
chore(footer): add Mastodon link

### LINKED ISSUE #9688: Add Mastodon icon link on the footer
Hello, this is my first time contribution to Yari! 🙂 

Currently, I'm going to create a PR that replaces the Twitter icon link with the newly created Mastodon icon link.

But I'm not sure this aligns with the MDN team's plan. Does the MDN team plan to continue using the Twitter account (https://twitter.com/mozdevnet) to post any other tweets if needed?

This is just my personal opinion, but I'm a bit afraid that keeping using that platform, which seems to be conflicting with MDN/Mozilla and our seeking idea of the Open Web, might end up sending a false message or giving an endorsement for that platform, to the open community.

However, I won't disagree if MDN still needs it for now. In that case, I'll soon revert the removal change and I plan to place a new Mastodon icon to the left of the existing Twitter icon. Please tell me which change is appropriate. Thank you.

### Validations

- [x] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [x] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) that reports the same bug to avoid creating a duplicate.
- [x] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions/categories/platform).
```[tasklist]
### Tasks
- [ ] Decide for design of new matodon icon with help of UX
```

### PR DESCRIPTION
**Pull request by shuuji3**
## Summary

Fixes #9688

### Problem

Recently, MDN launched the official Mastodon account https://mozilla.social/@MDN/111047293974029651 at mozilla.social (thanks for using Elk!). However, although I can see the link from https://github.com/mdn page, there is no Mastodon icon link on the MDN website. Discoverability is important for the Mastodon account.

### Solution

This PR replaces the Twitter icon link with the newly created Mastodon icon link.

But I'm not sure this is an appropriate change yet. Please see the comment on the original issue.

---

### Before

Screenshot 2023-09-19 at 4 27 09

### After

Screenshot 2023-09-19 at 3 09 25
Screenshot 2023-09-19 at 3 09 43
Screenshot 2023-09-19 at 3 38 51

---

### CODE REVIEW

**Comment by shuuji3** [comment]
I've updated it to include both social icon links on the footer. And I confirmed both links are working as expected on the local dev server.

Screenshot from 2023-10-11 22-17-30

File: client/src/ui/atoms/icon/index.scss [hunk 1]
```diff
@@ -8,7 +8,7 @@ $icons: "add-filled", "add", "altname", "bell-filled", "bell", "bookmark-filled"
   "prefix", "preview", "previous", "quote", "safari", "samsunginternet",
   "search", "send", "server", "sidebar", "simple-firefox", "small-arrow",
   "theme-light", "star-filled", "star", "theme-os-default", "thumbs-down",
-  "thumbs-up", "trash", "twitter", "unknown", "warning", "webview", "yes",
+  "thumbs-up", "trash", "twitter", "mastodon", "unknown", "warning", "webview", "yes",
```
Comment (caugner) [inline client/src/ui/atoms/icon/index.scss:11]: This list is (mostly) alphabetically ordered, can you add "mastodon" between "language" and "menu-filled", please?

**Review by caugner — CHANGES_REQUESTED** [review]
LGTM, just one nit (see my comment). 👍 

Also: Can you please add one screenshot for dark mode as well? Thanks!

File: client/src/ui/atoms/icon/index.scss (hunk 1 above)
Comment (shuuji3) [reply client/src/ui/atoms/icon/index.scss]: Thanks for your review 🙂

Moved the place of `"mastodon"` string.

**Comment by shuuji3** [comment]
Here are the latest screenshots.

## Large screen
Screenshot 2023-10-31 at 23 20 42
Screenshot 2023-10-31 at 23 20 55

## Mobile screen
Screenshot 2023-10-31 at 23 23 44
Screenshot 2023-10-31 at 23 21 16

**Review by caugner — APPROVED** [review]
LGTM, thank you! 🙌

**Comment by shuuji3** [comment]
Thank you! 🙂

**Pull request merged**